<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>测试报告</title>
    <link href="assets/style.css" rel="stylesheet" type="text/css"/></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function find_all(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sort_column(elem) {
    toggle_sort_states(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = key_result;
    } else if (elem.classList.contains('links')) {
        key = key_link;
    } else {
        key = key_alpha;
    }
    sort_table(elem, key(colIndex));
}

function show_all_extras() { // eslint-disable-line no-unused-vars
    find_all('.col-result').forEach(show_extras);
}

function hide_all_extras() { // eslint-disable-line no-unused-vars
    find_all('.col-result').forEach(hide_extras);
}

function show_extras(colresult_elem) {
    const extras = colresult_elem.parentNode.nextElementSibling;
    const expandcollapse = colresult_elem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hide_extras(colresult_elem) {
    const extras = colresult_elem.parentNode.nextElementSibling;
    const expandcollapse = colresult_elem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function show_filters() {
    const filter_items = document.getElementsByClassName('filter');
    for (let i = 0; i < filter_items.length; i++)
        filter_items[i].hidden = false;
}

function add_collapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:show_all_extras()">显示详情</a> / ' +
                            '<a href="javascript:hide_all_extras()">隐藏详情</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    find_all('.col-result').forEach(function(elem) {
        const collapsed = get_query_parameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                show_extras(event.currentTarget);
            } else {
                hide_extras(event.currentTarget);
            }
        });
    });
}

function get_query_parameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    reset_sort_headers();

    add_collapse();

    show_filters();

    toggle_sort_states(find('.initial-sort'));

    find_all('.sortable').forEach(function(elem) {
        elem.addEventListener("click",
                              function(event) {
                                  sort_column(elem);
                              }, false)
    });
	// 修改用例报告显示的用例名称
	var case_name_td = document.getElementsByClassName("col-name");
		for(var i = 0; i < case_name_td.length; i++)
			try{
				case_name_td[i].innerText = case_name_td[i].innerText.split("\[")[1].split("\]")[0];
			}
			catch(err){
				// 如果表格中没有[]会抛异常，如果抛异常我就显示null，如果你想显示别的东西自己改吧，因为通常只要我们使用参数化就有[]显示
				case_name_td[i].innerText = "测试用例无参数数据";
			}

};

function sort_table(clicked, key_func) {
    const rows = find_all('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sorted_rows = sort(rows, key_func, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sorted_rows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, key_func, reversed) {
    const sort_array = items.map(function(item, i) {
        return [key_func(item), i];
    });

    sort_array.sort(function(a, b) {
        const key_a = a[0];
        const key_b = b[0];

        if (key_a == key_b) return 0;

        if (reversed) {
            return key_a < key_b ? 1 : -1;
        } else {
            return key_a > key_b ? 1 : -1;
        }
    });

    return sort_array.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function key_alpha(col_index) {
    return function(elem) {
        return elem.childNodes[1].childNodes[col_index].firstChild.data.toLowerCase();
    };
}

function key_link(col_index) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[col_index].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function key_result(col_index) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[col_index].firstChild.data);
    };
}

function reset_sort_headers() {
    find_all('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    find_all('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggle_sort_states(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        reset_sort_headers();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function is_all_rows_hidden(value) {
    return value.hidden == false;
}

function filter_table(elem) { // eslint-disable-line no-unused-vars
    const outcome_att = 'data-test-result';
    const outcome = elem.getAttribute(outcome_att);
    const class_outcome = outcome + ' results-table-row';
    const outcome_rows = document.getElementsByClassName(class_outcome);

    for(let i = 0; i < outcome_rows.length; i++){
        outcome_rows[i].hidden = !elem.checked;
    }

    const rows = find_all('.results-table-row').filter(is_all_rows_hidden);
    const all_rows_hidden = rows.length == 0 ? true : false;
    const not_found_message = document.getElementById('not-found-message');
    not_found_message.hidden = !all_rows_hidden;
}
</script>
    <h1>signup.html</h1>
    <p>生成报告时间：2022-08-29 15:16:03</p>
    <h2>测试环境</h2>
    <table id="environment">
      <tr>
        <td>JAVA_HOME</td>
        <td>C:\Program Files\Java\jdk1.8.0_152</td></tr>
      <tr>
        <td>Packages</td>
        <td>{"pluggy": "1.0.0", "py": "1.11.0", "pytest": "7.1.2"}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Windows-10-10.0.19041-SP0</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{"html": "3.1.1", "metadata": "2.0.2"}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.8.1</td></tr></table>
    <h2>用例统计</h2>
    <p>执行了12个测试，耗时1.15秒</p>
    <p class="filter" hidden="true">(取消)勾选复选框, 以便筛选测试结果</p><input checked="true" class="filter" data-test-result="passed" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="passed">10 通过</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="skipped">0 跳过</span>, <input checked="true" class="filter" data-test-result="failed" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="failed">2 失败</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="error">0 错误</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xfailed">0 预期失败</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xpassed">0 预期通过</span>
    <h2>测试结果</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">通过/失败</th>
          <th class="sortable" col="name">测试用例</th>
          <th class="sortable" col="duration">持续时间</th>
          <th class="sortable links" col="links">链接</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">无测试结果, 试着选择其他测试结果条件</th></tr></thead>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">signup.py::test_signup[signup_04-测试注册确认密码为空-http://192.168.47.132:80/apitest/signup/-post-json-args3-expect3-users,info-select count(*), (select count(*) from info where name=&apos;注册确认密码为空用户&apos;) from users where username=&apos;test08&apos;-db_expect3]</td>
          <td class="col-duration">0.03</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">case_info = &#x27;signup_04-测试注册确认密码为空&#x27;<br/>url = &#x27;http://192.168.47.132:80/apitest/signup/&#x27;, method = &#x27;post&#x27;<br/>args_type = &#x27;json&#x27;<br/>args = {&#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;username&#x27;: &#x27;test08&#x27;}<br/>expect = {&#x27;Message&#x27;: &#x27;注册失败，确认密码不能为空&#x27;, &#x27;Result&#x27;: &#x27;Confirm password is null&#x27;, &#x27;Status&#x27;: 1003}<br/>tables = &#x27;users,info&#x27;<br/>check_sql = &quot;select count(*), (select count(*) from info where name=&#x27;注册确认密码为空用户&#x27;) from users where username=&#x27;test08&#x27;&quot;<br/>db_expect = (0, 0)<br/><br/>    @pytest.mark.parametrize(&#x27;case_info, url, method, args_type, args, expect, tables, check_sql, db_expect&#x27;, cases)<br/>    def test_signup(case_info, url, method, args_type, args, expect, tables, check_sql, db_expect):<br/>        actual = send_request(url, method, args_type, args)<br/>        passed, msg = check(case_info, actual, expect)<br/>        try:<br/>            assert passed, msg<br/>        finally:<br/>            passed, msg = DB().check_db(case_info, tables, args, check_sql, db_expect)<br/>&gt;           assert passed, msg<br/><span class="error">E           AssertionError: signup_04-测试注册确认密码为空==验库失败，检查表users,info中，参数为{&#x27;username&#x27;: &#x27;test08&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;}的</span><br/><span class="error">E             	预期(0, 0)</span><br/><span class="error">E             	实际(1, 1)语句</span><br/><span class="error">E           assert False</span><br/><br/>signup.py:20: AssertionError<br/> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 103
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test08&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;})成功
INFO:root:signup_04-测试注册确认密码为空+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
WARNING:root:signup_04-测试注册确认密码为空==验库失败，检查表users,info中，参数为{&#x27;username&#x27;: &#x27;test08&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;}的
	预期(0, 0)
	实际(1, 1)语句
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 103
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test08&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;})成功
INFO     root:case.py:41 signup_04-测试注册确认密码为空+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
WARNING  root:db.py:61 signup_04-测试注册确认密码为空==验库失败，检查表users,info中，参数为{&#x27;username&#x27;: &#x27;test08&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;注册确认密码为空用户&#x27;}的
	预期(0, 0)
	实际(1, 1)语句<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">signup.py::test_signup[signup_11-测试注册密码与确认密码不一致-http://192.168.47.132:80/apitest/signup/-post-json-args10-expect10-users,info-select count(*), (select count(*) from info where name=&apos;注册两个密码不一致用户&apos;) from users where username=&apos;test14&apos;-db_expect10]</td>
          <td class="col-duration">0.04</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">case_info = &#x27;signup_11-测试注册密码与确认密码不一致&#x27;<br/>url = &#x27;http://192.168.47.132:80/apitest/signup/&#x27;, method = &#x27;post&#x27;<br/>args_type = &#x27;json&#x27;<br/>args = {&#x27;confirm&#x27;: &#x27;123&#x27;, &#x27;name&#x27;: &#x27;注册两个密码不一致用户&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;username&#x27;: &#x27;test14&#x27;}<br/>expect = {&#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;, &#x27;Result&#x27;: &#x27;Two passwords not compare&#x27;, &#x27;Status&#x27;: 1009}<br/>tables = &#x27;users,info&#x27;<br/>check_sql = &quot;select count(*), (select count(*) from info where name=&#x27;注册两个密码不一致用户&#x27;) from users where username=&#x27;test14&#x27;&quot;<br/>db_expect = (0, 0)<br/><br/>    @pytest.mark.parametrize(&#x27;case_info, url, method, args_type, args, expect, tables, check_sql, db_expect&#x27;, cases)<br/>    def test_signup(case_info, url, method, args_type, args, expect, tables, check_sql, db_expect):<br/>        actual = send_request(url, method, args_type, args)<br/>        passed, msg = check(case_info, actual, expect)<br/>        try:<br/>&gt;           assert passed, msg<br/><span class="error">E           AssertionError: signup_11-测试注册密码与确认密码不一致==响应比对失败，</span><br/><span class="error">E             	预期{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Two passwords not compare&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}</span><br/><span class="error">E             	实际{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Confirm password error&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}</span><br/><span class="error">E           assert False</span><br/><br/>signup.py:17: AssertionError<br/> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 98
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test14&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123&#x27;, &#x27;name&#x27;: &#x27;注册两个密码不一致用户&#x27;})成功
WARNING:root:signup_11-测试注册密码与确认密码不一致==响应比对失败，
	预期{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Two passwords not compare&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}
	实际{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Confirm password error&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_11-测试注册密码与确认密码不一致语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 98
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test14&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123&#x27;, &#x27;name&#x27;: &#x27;注册两个密码不一致用户&#x27;})成功
WARNING  root:case.py:45 signup_11-测试注册密码与确认密码不一致==响应比对失败，
	预期{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Two passwords not compare&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}
	实际{&#x27;Status&#x27;: 1009, &#x27;Result&#x27;: &#x27;Confirm password error&#x27;, &#x27;Message&#x27;: &#x27;注册失败，两个密码不一致&#x27;}
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_11-测试注册密码与确认密码不一致语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_01-测试成功注册-http://192.168.47.132:80/apitest/signup/-post-json-args0-expect0-users,info-select count(*), (select count(*) from info where name=&apos;成功注册用户&apos; and id=users.id) from users where username=&apos;test06&apos; and password=md5(&apos;123456&apos;)-db_expect0]</td>
          <td class="col-duration">0.03</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 59
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test06&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;成功注册用户&#x27;})成功
INFO:root:signup_01-测试成功注册+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_01-测试成功注册语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 59
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test06&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;成功注册用户&#x27;})成功
INFO     root:case.py:41 signup_01-测试成功注册+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_01-测试成功注册语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_02-测试注册用户名为空-http://192.168.47.132:80/apitest/signup/-post-json-args1-expect1-users,info-select count(*), (select count(*) from info where name=&apos;注册用户名为空用户&apos;) from users where username=&apos;&apos;-db_expect1]</td>
          <td class="col-duration">0.03</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 92
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名为空用户&#x27;})成功
INFO:root:signup_02-测试注册用户名为空+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_02-测试注册用户名为空语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 92
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名为空用户&#x27;})成功
INFO     root:case.py:41 signup_02-测试注册用户名为空+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_02-测试注册用户名为空语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_03-测试注册密码为空-http://192.168.47.132:80/apitest/signup/-post-json-args2-expect2-users,info-select count(*), (select count(*) from info where name=&apos;注册密码为空用户&apos;) from users where username=&apos;test07&apos;-db_expect2]</td>
          <td class="col-duration">0.03</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 89
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test07&#x27;, &#x27;password&#x27;: &#x27;&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册密码为空用户&#x27;})成功
INFO:root:signup_03-测试注册密码为空+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_03-测试注册密码为空语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 89
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test07&#x27;, &#x27;password&#x27;: &#x27;&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册密码为空用户&#x27;})成功
INFO     root:case.py:41 signup_03-测试注册密码为空+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_03-测试注册密码为空语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_05-测试注册姓名为空-http://192.168.47.132:80/apitest/signup/-post-json-args4-expect4-users,info-select count(*), (select count(*) from info where name=&apos;&apos;) from users where username=&apos;test09&apos;-db_expect4]</td>
          <td class="col-duration">0.02</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 85
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test09&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;&#x27;})成功
INFO:root:signup_05-测试注册姓名为空+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_05-测试注册姓名为空语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 85
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test09&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;&#x27;})成功
INFO     root:case.py:41 signup_05-测试注册姓名为空+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_05-测试注册姓名为空语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_06-测试注册所有参数为空-http://192.168.47.132:80/apitest/signup/-post-json-args5-expect5-users,info-select count(*), (select count(*) from info where name=&apos;&apos;) from users where username=&apos;&apos;-db_expect5]</td>
          <td class="col-duration">0.03</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 92
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;&#x27;, &#x27;password&#x27;: &#x27;&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;&#x27;})成功
INFO:root:signup_06-测试注册所有参数为空+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_06-测试注册所有参数为空语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 92
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;&#x27;, &#x27;password&#x27;: &#x27;&#x27;, &#x27;confirm&#x27;: &#x27;&#x27;, &#x27;name&#x27;: &#x27;&#x27;})成功
INFO     root:case.py:41 signup_06-测试注册所有参数为空+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_06-测试注册所有参数为空语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_07-测试注册用户名字数超过30-http://192.168.47.132:80/apitest/signup/-post-json-args6-expect6-users,info-select count(*), (select count(*) from info where name=&apos;注册用户名31个字用户&apos;) from users where username=&apos;test10test10test10test10test10-&apos;-db_expect6]</td>
          <td class="col-duration">0.02</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 120
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test10test10test10test10test10-&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名31个字用户&#x27;})成功
INFO:root:signup_07-测试注册用户名字数超过30+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_07-测试注册用户名字数超过30语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 120
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test10test10test10test10test10-&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名31个字用户&#x27;})成功
INFO     root:case.py:41 signup_07-测试注册用户名字数超过30+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_07-测试注册用户名字数超过30语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_08-测试注册密码字数超过50-http://192.168.47.132:80/apitest/signup/-post-json-args7-expect7-users,info-select count(*), (select count(*) from info where name=&apos;注册密码51个字用户&apos;) from users where username=&apos;test11&apos;-db_expect7]</td>
          <td class="col-duration">0.02</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 117
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test11&#x27;, &#x27;password&#x27;: &#x27;123456789012345678901234567890123456789012345678901&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册密码51个字用户&#x27;})成功
INFO:root:signup_08-测试注册密码字数超过50+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_08-测试注册密码字数超过50语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 117
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test11&#x27;, &#x27;password&#x27;: &#x27;123456789012345678901234567890123456789012345678901&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册密码51个字用户&#x27;})成功
INFO     root:case.py:41 signup_08-测试注册密码字数超过50+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_08-测试注册密码字数超过50语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_09-测试注册确认密码字数超过50-http://192.168.47.132:80/apitest/signup/-post-json-args8-expect8-users,info-select count(*), (select count(*) from info where name=&apos;注册确认密码51个字用户&apos;) from users where username=&apos;test12&apos;-db_expect8]</td>
          <td class="col-duration">0.02</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 122
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test12&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456789012345678901234567890123456789012345678901&#x27;, &#x27;name&#x27;: &#x27;注册确认密码51个字用户&#x27;})成功
INFO:root:signup_09-测试注册确认密码字数超过50+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_09-测试注册确认密码字数超过50语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 122
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test12&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456789012345678901234567890123456789012345678901&#x27;, &#x27;name&#x27;: &#x27;注册确认密码51个字用户&#x27;})成功
INFO     root:case.py:41 signup_09-测试注册确认密码字数超过50+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_09-测试注册确认密码字数超过50语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_10-测试注册姓名字数超过20-http://192.168.47.132:80/apitest/signup/-post-json-args9-expect9-users,info-select count(*), (select count(*) from info where name=&apos;注册姓名字数21用户-注册姓名字数21用户&apos;) from users where username=&apos;test13&apos;-db_expect9]</td>
          <td class="col-duration">0.02</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 113
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test13&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册姓名字数21用户-注册姓名字数21用户&#x27;})成功
INFO:root:signup_10-测试注册姓名字数超过20+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_10-测试注册姓名字数超过20语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 113
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test13&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册姓名字数21用户-注册姓名字数21用户&#x27;})成功
INFO     root:case.py:41 signup_10-测试注册姓名字数超过20+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_10-测试注册姓名字数超过20语句验库成功<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">signup.py::test_signup[signup_12-测试用户名被占用-http://192.168.47.132:80/apitest/signup/-post-json-args11-expect11-users,info-select count(*), (select count(*) from info where name=&apos;注册用户名已存在用户&apos; and id=users.id) from users where username=&apos;test15&apos; and password=md5(&apos;123456&apos;) and regtime=&apos;2022/08/01&apos;-db_expect11]</td>
          <td class="col-duration">0.04</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> ------------------------------Captured stderr call------------------------------ <br/>DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.47.132:80
DEBUG:urllib3.connectionpool:http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 100
DEBUG:root:请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test15&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名已存在用户&#x27;})成功
INFO:root:signup_12-测试用户名被占用+++通过++
DEBUG:root:读配置文件../conf/server.conf成功
DEBUG:root:获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG:root:数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO:root:数据库执行signup_12-测试用户名被占用语句验库成功
<br/> -------------------------------Captured log call-------------------------------- <br/>DEBUG    urllib3.connectionpool:connectionpool.py:228 Starting new HTTP connection (1): 192.168.47.132:80
DEBUG    urllib3.connectionpool:connectionpool.py:456 http://192.168.47.132:80 &quot;POST /apitest/signup/ HTTP/1.1&quot; 200 100
DEBUG    root:case.py:32 请求地址requests.post(&#x27;http://192.168.47.132:80/apitest/signup/&#x27;,json={&#x27;username&#x27;: &#x27;test15&#x27;, &#x27;password&#x27;: &#x27;123456&#x27;, &#x27;confirm&#x27;: &#x27;123456&#x27;, &#x27;name&#x27;: &#x27;注册用户名已存在用户&#x27;})成功
INFO     root:case.py:41 signup_12-测试用户名被占用+++通过++
DEBUG    root:conf.py:17 读配置文件../conf/server.conf成功
DEBUG    root:conf.py:46 获取数据库信息{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
DEBUG    root:db.py:33 数据库连接成功{&#x27;host&#x27;: &#x27;192.168.47.132&#x27;, &#x27;db&#x27;: &#x27;wa_test&#x27;, &#x27;user&#x27;: &#x27;root&#x27;, &#x27;passwd&#x27;: &#x27;123456&#x27;}成功
INFO     root:db.py:58 数据库执行signup_12-测试用户名被占用语句验库成功<br/></div></td></tr></tbody></table></body></html>